pygin/pygin/src/helpers.hpp

```diff
@@ -45,4 +46,30 @@ namespace helpers
 			return Result;
 		}
 	}
+
+	class py_string_storage
+	{
+	public:
+		py_string_storage() {}
+
+		py_string_storage(py_string_storage const&) = delete;
+		py_string_storage& operator= (py_string_storage const&) = delete;
+
+		~py_string_storage(){ this->clear(); }
+
+		void reserve(size_t size) { this->strings.reserve(size); }
```

Comment:
Please don't use explicit `this` where it's not necessary. `m_` prefix is preferable.

Reply:
Fixed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.